blog/resoto-query-language-aggregation/index.md

```diff
@@ -3,16 +3,19 @@ authors: [matthias]
 tags: [query language, aws, gcp]
 ---
 
-# Aggregating data with Resoto
+# Aggregating Data with Resoto
 
-This post is the second episode of the Resoto query language series. While it is not required, I would encourage you to read the [RQL 101](/blog/2022/02/04/resoto-query-language-101) post to get a short
-introduction into the capabilities of the Resoto query language.
+:::note
 
-RQL allows selecting resources using filters, combinators, and traversals. It is probably the primary use case to perform a query and retrieve the resulting list of resources. This list can be used in other actions and commands or serves information retrieval. While it is beneficial to get a list of resource data,  we also want to combine, group, and aggregate data about our resources.
+This blog post is the second in a series about Resoto's powerful query language. The previous post, [Resoto Query Language 101](/blog/2022/02/04/resoto-query-language-101), provides an introduction to the query language.
```

Comment:
```suggestion
This blog post is the second in a series about Resoto's powerful search functionality. The previous post, [Resoto Search 101](/blog/2022/02/04/resoto-search-101), provides an introduction to Resoto's search capabilities.
```

Reply:
You'll want to rebase this branch to `main` so that Docusaurus doesn't think this is a broken link (the build will fail on broken internal links).